packages/sdk-metrics/package.json

```diff
@@ -79,7 +79,7 @@
     "webpack-merge": "5.10.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.3.0 <1.10.0"
+    "@opentelemetry/api": ">=1.9.0 <1.10.0"
```

Comment:
Not sure what the pessimistic upper bound is about, kept it the same cause I figured it would a different conversation to change that.

Reply:
This is due to the fact that API minor versions may be breaking for implementations of it's interfaces in minor versions. `1.10.0` may require us to implement new interfaces, so we intentionally do not support future API versions.

This is something that's common across all language implementations of OTel - see 

https://github.com/open-telemetry/opentelemetry-specification/blob/a265ae0628177be25dc477ea8fe200f1c825b871/specification/upgrading.md?plain=1#L75-L89

and 

https://github.com/open-telemetry/opentelemetry-specification/blob/a265ae0628177be25dc477ea8fe200f1c825b871/specification/upgrading.md?plain=1#L106-L109